gulpfile.js

```diff
@@ -11,7 +11,8 @@ var version = require('./package.json').version;
 function compile() {
   return gulp.src('lib/*.es6')
     .pipe(babel({
-      experimental: true
+      experimental: true,
+      loose: ['es6.modules', 'es6.classes']
```

Comment:
turned out that loose mode for es6.modules and es6.classes are all that's necessary. the last bit required a change in the test file and wasn't babel related.